central_v2/src/lib/ScreenEnums.tsx

```diff
@@ -0,0 +1,10 @@
+export enum Screen {
```

Comment:
I can tell what this is because of the options but a more descriptive name for this `enum` might improve readability when we the code expands

Reply:
SelectedCentralPages